string name
---
bool success
string message
float64 x
float64 y
float64 z

